commit 6065e18a502a7385e857bfdbf0317538c97225c8
Author: Cameron Gutman <aicommander@gmail.com>
Date:   Sat Dec 26 21:27:18 2020 -0600

    Do not enable OpenGL texture compatibility in VT
    
    This causes severe decoder performance impacts on Big Sur at 4K.

diff --git a/libavcodec/videotoolbox.c b/libavcodec/videotoolbox.c
index 8773de33930..c9d775d82b1 100644
--- a/libavcodec/videotoolbox.c
+++ b/libavcodec/videotoolbox.c
@@ -700,11 +700,6 @@ static CFDictionaryRef videotoolbox_buffer_attributes_create(int width,
     CFDictionarySetValue(buffer_attributes, kCVPixelBufferIOSurfacePropertiesKey, io_surface_properties);
     CFDictionarySetValue(buffer_attributes, kCVPixelBufferWidthKey, w);
     CFDictionarySetValue(buffer_attributes, kCVPixelBufferHeightKey, h);
-#if TARGET_OS_IPHONE
-    CFDictionarySetValue(buffer_attributes, kCVPixelBufferOpenGLESCompatibilityKey, kCFBooleanTrue);
-#else
-    CFDictionarySetValue(buffer_attributes, kCVPixelBufferIOSurfaceOpenGLTextureCompatibilityKey, kCFBooleanTrue);
-#endif
 
     CFRelease(io_surface_properties);
     CFRelease(cv_pix_fmt);
